alica_tests/src/test/Expr/src/Behaviour/ConstraintUsingBehaviour.cpp

```diff
@@ -37,16 +37,17 @@ void ConstraintUsingBehaviour::run(void* msg)
     if (result.size() > 0) {
         return;
     }
-    _query.getSolution<reasoner::ConstraintTestPlanDummySolver, int64_t>(getPlanContext(), result);
+    _query->getSolution<reasoner::ConstraintTestPlanDummySolver, int64_t>(getPlanContext(), result);
     /*PROTECTED REGION END*/
 }
 void ConstraintUsingBehaviour::initialiseParameters()
 {
     /*PROTECTED REGION ID(initialiseParameters1414068597716) ENABLED START*/
     // Add additional options here
     _callCounter = 0;
-    _query.clearStaticVariables();
-    _query.addStaticVariable(getVariable("Y"));
+    _query = std::make_unique<Query>(getLogger());
```

Comment:
fix: no need to take in the logger

Reply:
fixed in https://github.com/rapyuta-robotics/alica/pull/302